### DISCUSSION TITLE
Adding .NET Code Coverage to your Azure DevOps pipeline

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#360] [Announcements]
# Adding .NET Code Coverage to your Azure DevOps pipeline

A step-by-step guide on how to add code coverage to your .NET project in Azure DevOps.

https://timdeschryver.dev/blog/adding-net-code-coverage-to-your-azure-devops-pipeline

**Comment by mattfrear** [comment]
Solid post, thank you.

I recently spent a day or so figuring out how this works by myself. Wish I'd found this post sooner, it would have saved me some time.

**Reply by timdeschryver** [reply]
Thanks @mattfrear , it also took a moment to get this right 😅
